### ISSUE TITLE
Uncaught TypeError: this.callback.call is not a function

### ISSUE THREAD

**Issue by RoopGuron** [PredixDev/px-data-table#36]
Upon using px-data-table component i am getting the following error.
In one of the components, the following code...
   ` <px-data-table table-data="{{data}}"></px-data-table>` 
with the data property set to be...
```
[{
                "client_id" : "foo",
                "name" : "Foo Client Name",
                "scope" : ["uaa.none"],
                "resource_ids" : ["none"],
                "authorities" : ["cloud_controller.read","cloud_controller.write","scim.read"],
                "authorized_grant_types" : ["client_credentials"],
                "lastModified" : 1426260091149
              },
                {
                  "client_id" : "bar",
                  "name" : "Bar Client Name",
                  "scope" : ["cloud_controller.read","cloud_controller.write","openid"],
                  "resource_ids" : ["none"],
                  "authorities" : ["uaa.none"],
                  "authorized_grant_types" : ["authorization_code"],
                  "lastModified" : 1426260091145
                }];
```
is causing the following error:
**_Uncaught TypeError: this.callback.call is not a function_**

![image](https://cloud.githubusercontent.com/assets/10160285/15478170/aa53142c-20cd-11e6-9169-a5d5874a0881.png)

**Comment by mdwragg** [comment]
Hi, your data format is incorrect. The correct format is an array of objects. Each object is a 'row' in the table containing key (column name) - value (cell value) pairs.

The arrays in the 'value' will be causing your issue here:

`  "authorities": ["cloud_controller.read", "cloud_controller.write", "scim.read"],`

These would need to be flattened before attempting to binding to the table element.

**Comment by GE-RossRothenstine** [comment]
This should be resolved with Pull Request #47 .
